tests/fastwarc/test_warc.py

```diff
@@ -8,6 +8,7 @@
 import lz4.frame
 import io
 import os
+import hashlib
```

Comment:
For example for this test line (https://github.com/chatnoir-eu/chatnoir-resiliparse/blob/774e33f45b5aef34fade03b465888925169e76b5/tests/fastwarc/test_warc.py#L330) the import was missing. How did this work previously?

Reply:
This change is unnecessary. There's already an `import hashlib` below the `import gzip` due to the rebase.